### PR TITLE
Fix some typos (found by codespell)

### PR DESCRIPTION
**Pull request by stweil**
## Proposed changes

These commits fix typos in various parts of the code and documentation (ignoring code from vendors).

## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [x] I have squashed my commits into logic units.
- [x] I have described the changes in the commit messages.

## Other information

The files for localization (*.po) still have to be updated.
Maybe add codespell to the continuous integration tests?

### CODE REVIEW

File: docs/admin/install.rst [hunk 1]
```diff
@@ -142,7 +142,7 @@ The following dependencies have to be installed on the system:
 
 ``Git``
     https://git-scm.com/
-Pango, Cairo and related header files and gir introspection data
+Pango, Cairo and related header files and git introspection data
```
Comment (nijel) [inline docs/admin/install.rst:145]: gir stands for [GObject Introspection](https://en.wikipedia.org/wiki/GObject#GObject_Introspection) here:

```suggestion
Pango, Cairo and related header files and GObject introspection data
```

File: docs/devel/html.rst [hunk 2]
```diff
@@ -93,7 +93,7 @@ selector to process all elements.
 .. note::
 
    Right now, only text of the elements is extracted. This add-on doesn't support localization
-   of element attributes or elements with childs.
+   of element attributes or elements with children.
```
Comment (orangesunny) [inline docs/devel/html.rst:96]: This does not make sense. A different word should be chosen, IMO. Something with sub- ?

File: weblate/formats/convert.py [hunk 3]
```diff
@@ -240,12 +240,12 @@ def convertfile(storefile, template_store):
 
     def save_content(self, handle):
         """Store content to file."""
-        convertor = po2html()
+        converter = po2html()
```
Comment (orangesunny) [inline weblate/formats/convert.py:243]: @nijel, I am afraid that this could break the code?

File: weblate/wladmin/views.py [hunk 4]
```diff
@@ -460,23 +460,23 @@ def billing(request):
     # We will list all billings anyway, so fetch  them at once
     billings = Billing.objects.prefetch().order_by("expiry", "removal", "id")
 
-    for currrent in billings:
-        if currrent.removal:
-            removal.append(currrent)
-        elif currrent.state == Billing.STATE_TRIAL:
+    for current in billings:
```
Comment (orangesunny) [inline weblate/wladmin/views.py:463]: @nijel, this is correct, but I am unsure if it wouldn’t break anything.

**Review by orangesunny — CHANGES_REQUESTED** [review]
Can go if my comments are solved.

File: docs/admin/install.rst [hunk 5]
```diff
@@ -142,7 +142,7 @@ The following dependencies have to be installed on the system:
 
 ``Git``
     https://git-scm.com/
-Pango, Cairo and related header files and gir introspection data
```
[the remark this answers is not in the document]
Comment (nijel) [reply docs/admin/install.rst:145]: I've made this change, it is correct.

File: docs/devel/html.rst (hunk 2 above)
Comment (nijel) [reply docs/devel/html.rst:96]: child elements is established in XML, so it looks okay to me.

File: weblate/formats/convert.py (hunk 3 above)
Comment (nijel) [reply weblate/formats/convert.py:243]: Tests pass, so it should be okay :-).

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**